pytorch_lightning/loggers/neptune.py

```diff
@@ -284,9 +538,7 @@ def name(self) -> str:
         Returns:
             The name of the experiment if not in offline mode else "offline-name".
         """
-        if self.offline_mode:
-            return "offline-name"
-        return self.experiment.name
+        return "NeptuneLogger"
```

Comment:
Why `NeptuneLogger` there ? This is a BC.

Reply:
I think we should return the actual experiment name here by querying the client